piet-web/src/text.rs

```diff
@@ -63,6 +63,10 @@ impl Text for WebText {
         Some(FontFamily::new_unchecked(family_name))
     }
 
+    fn load_font(&mut self, _data: &[u8]) -> Result<FontFamily, Error> {
```

Comment:
I think this could be added, though, at considerable difficulty? Probably deserves some thought, maybe an issue.

Reply:
yes certainly *likely* possible; I'm not treating web as a 'tier 1' target for the purposes of this current work though.